drone-fly-integration-tests/src/test/java/com/expediagroup/dataplatform/dronefly/core/integration/DroneFlyIntegrationTestUtils.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.dataplatform.dronefly.core.integration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import com.google.common.collect.Lists;
+
+public class DroneFlyIntegrationTestUtils {
+
+  static final String TOPIC = "apiary-events";
+  static final String DATABASE = "database";
+  static final String TABLE = "table";
+
+  public static Table buildTable() {
+    return buildTable(null);
+  }
+
+  public static Table buildTable(String tableName) {
+    List<FieldSchema> partitions = Lists.newArrayList();
+    partitions.add(new FieldSchema("a", "string", "comment"));
+    partitions.add(new FieldSchema("b", "string", "comment"));
+    partitions.add(new FieldSchema("c", "string", "comment"));
+    return new Table(tableName == null ? TABLE : tableName, DATABASE, "me", 1, 1, 1, new StorageDescriptor(),
+        partitions, buildTableParameters(), "originalText", "expandedText", "tableType");
+  }
+
+  public static Partition buildPartition() {
+    return buildPartition(null);
```

Comment:
Is it OK to pass null in and have this used as the partition name below?

Reply:
I have changed it to a string "partition" so it doesn't look weird.